doc/src/modules/codegen.rst

```diff
@@ -0,0 +1,539 @@
+===============
```

Comment:
Note that there is a page called 'codegen" already that focused just on the code generation class:

http://docs.sympy.org/latest/modules/utilities/codegen.html

The above page is strictly printing the documentation for that one modue: codegen.

What can we do to not have two pages called the same thing? There should at least be some cross referencing. Is there a better name for this page because it includes things about printing, code generation, and automatic wrapping? This made me realize why I request this content be in the "Numerical Computation" page, because it seems like a better title.

Maybe the bigger issue is that our docs currently mix prose documentation with API reference and there is no clear place for documentation that encompasses information from multiple modules.